### DISCUSSION TITLE
Feature: Add ability for eg. sleep commands in the recording

### DISCUSSION THREAD

**Discussion by sessl3r** [asciinema/discussions#95] [Ideas]
I used this tool to record some logging output from which I do a video afterwards.
In this video I want to demonstrate things so I need to "pause" the playback (which is possible manually I know).
But what would be great to have is a way to code something like a sleep into the cast-File (manually - after the recording).

For my own purpose I hacked this in (really ugly but works...): https://github.com/sessl3r/asciinema/commit/9a7f4b96d883bc30e22f117f2a898003ec380a4c

A example of a recording would look like:
```
[...]
[0.0, "o", "$ # This will do something cool"]
[0.1, "o", "\r\n"]
[0.2, "o", "$ whoami \r\n"]
[5.0, "sleep", ""]
[0.3, "o", "root \r\n"]
```

In the playback the second line would "pause" the playback for 5.0 seconds and for example in a video one could say something about this etc.

**Comment by ku1ik** [comment]
Your idea seems to be very similar to my idea of breakpoints, which I started working on in asciinema/asciinema#341. Gotta rebase that old PR and finish it finally though :sweat_smile:
